frontend/src/components/ParkDetailsSeasonDates.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from "prop-types";
+import { useEffect, useMemo } from "react";
+import useDate from "@/hooks/useDate";
+import DateRange from "@/components/DateRange";
+
+function DateTypeRow({ dateType }) {
+  console.log("dateType", dateType);
+
+  return (
+    <tr>
+      <td>{dateType.dateType.name} dates</td>
+      <td>
+        <DateRange start={dateType.startDate} end={dateType.endDate} />
+      </td>
+    </tr>
+  );
+}
+
+// "Dateable" features: Campsite groupings, etc.
+function CampGroundFeature({ feature }) {
+  return (
+    <div className="feature">
+      <h5>{feature.name}</h5>
+
+      <table className="table table-striped sub-area-dates mb-0">
+        <tbody>
+          {feature?.dateable?.currentSeasonDates?.map((dateType) => (
+            <DateTypeRow key={dateType.id} dateType={dateType} />
```

Comment:
The data from the API is not grouped by dateType. It would have a list of dates for the current season with both reservation and operation dates included there. So, a filter is probably needed for each type.

Reply:
If the placeholder data I put into the seasons.js endpoint is still accurate, then I'll group it on the frontend 👍 

I added a few extra dates to simulate a campsite grouping with 3 sets of dates so I'll update that.

Reply:
Pushed some updates to that file, now it will render multiple DateRange components in a single DateTypeRow if there are more than one